spec/expression.dd

```diff
@@ -507,18 +507,25 @@ $(GNAME EqualExpression):
         Otherwise, the bit patterns of the common type are compared for equality.
     )
 
-    $(P For complex numbers, equality is defined as equivalent to:
+    $(P For complex numbers, equality is defined as equivalent to:)
 
```

Comment:
We have deprecated built-in complex numbers, perhaps this sentence should be reworded or removed?

Reply:
I've indicated this now.